netweaver/extract_installer.sls

```diff
@@ -0,0 +1,12 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+{% set host = grains['host'] %}
+{% for node in netweaver.nodes if node.host == host and (netweaver.sapcar_exe_file is defined and netweaver.swpm_sar_file and netweaver.swpm_extract_dir is defined) %}
+
+extract_installer_file_{{ node.host }}:
+    netweaver.installer_file_extracted:
+    - sapcar_exe_file: {{ node.install.sapcar_exe_file }}
+    - swpm_sar_file: {{ node.install.swpm_sar_file }}
+    - swpm_extract_dir: {{ netweaver.swpm_extract_dir }}
```

Comment:
Make sure before running this method that the `outpur_dir` exists. You will need to add a new `file.directory` state before I guess

Reply:
And besides that, we need to make sure that the SAPCAR has executables rights, `755` file permissions work, but I don't know if there is some recommendation for that.
So, we will need to make sure that the file is executable before running the new methods.

Reply:
Other things. I have find out that the SIGNATURE.SMF is created in the folder where SAPCAR is executed (or -R folder), but I would say thay we will need to copy the created file to the folder where the installer is located, so we will need to create an additional state step to copy the file

Reply:
Hi @arbulu89 the `outpur_dir` folder gets created by sapcar if it doesnt exist & overridden if exists.
I have added step to check the sapcar permissions.

Reply:
Regarding the `SIGNATURE.SMF` incase of NW SWPM installer, this is extracted correctly to the location where `sapinst` is present.
We would need to copy that in case of HANA installer, maybe in the HANA formula that can be done.